### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by jonathantneal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Marble
```
Comment (zenorocha) [inline README.md]: WeDeploy.com

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+The local server will be available at [http://0.0.0.0:3001](http://0.0.0.0:3001).
+
+---
+
+To work on the deeper pages, which include blog, docs, etc.;
+
+1. From the [node/electric] directory, install the local dependencies:
+
+   ```sh
+   npm rium
```
Comment (zenorocha) [inline README.md]: npm run

**Comment by zenorocha** [comment]
Can you remove this readme as well? https://github.com/wedeploy/wedeploy.com/tree/master/node/electric

**Comment by jonathantneal** [comment]
Thanks, @zenorocha, @pragmaticivan! I’ve fixed the typo, added a badge for CI, added a badge for Slack, replaced the blockquote with more a meaningful description of the repository and WeDeploy, and then removed the README.md file within electric. I’ve also tried to generally make it look very attractive.

Would you please re-review?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+   ```
+
+The local server will be available at
+[http://localhost:8888](http://localhost:8888).
+
+[wedeploy]: https://wedeploy.com/
+[cli-url]: https://ci.wedeploy.com/job/wedeploy/job/wedeploy.com/job/master/
+[cli-img]: https://ci.wedeploy.com/buildStatus/icon?job=wedeploy/wedeploy.com/master
+[irc-url]: https://chat.wedeploy.com/
+[irc-img]: https://img.shields.io/badge/chat-slack-blue.svg
```
Comment (zenorocha) [inline README.md]: Use this one instead: https://chat.wedeploy.com/badge.svg

Comment (jonathantneal) [reply README.md]: Done.

**Pull request merged**